full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import { IModelConnection, ScreenViewport, SnapshotConnection, ViewCreator3d} from "@itwin/core-frontend";
+import { TestUtility } from "../TestUtility";
+import { ProcessDetector } from "@itwin/core-bentley";
+
+describe("ViewCreator3d", async () => {
+  let imodel: IModelConnection;
+
+  before(async () => {
+    await TestUtility.startFrontend();
+    imodel = await SnapshotConnection.openFile("mirukuru.ibim");
+  });
+
+  after(async () => {
+    await imodel?.close();
+    await TestUtility.shutdownFrontend();
+  });
+
+  it("should generate tiles when using a viewstate created by viewcreator3d", async () => {
+    const div = document.createElement("div");
+    div.style.width = div.style.height = "20px";
+    document.body.appendChild(div);
+    const viewcreator3d = new ViewCreator3d(imodel);
+    const viewState = await viewcreator3d.createDefaultView();
+    expect(viewState).to.exist.and.be.not.empty;
+
+    const testVp: ScreenViewport = ScreenViewport.create(div, viewState);
+    await testVp.waitForSceneCompletion();
+    if (ProcessDetector.isElectronAppFrontend) {
```

Comment:
just curious, why is this different between electron and browser?

Reply:
I don't have a good technical explanation for that, it seemed consistent atleast. Slight differences between rendering tiles in electron vs browser? Maybe Paul or mark would be able to explain.

Reply:
@pmconne @markschlosseratbentley ?

I wont hold the pr for this; but just curious to know

Reply:
I tried 400px and my test still passed (meaning electron was 7 tiles still). It also passed at 800px. I am running build:ci (almost fell into the trap of just running build).

Reply:
I guess I'll pull your branch and see what's up with the discrepancy.

Reply:
Fixed.